l10n_it_fatturapa_out_taxes/wizard/wizard_export_fatturapa.py

```diff
@@ -0,0 +1,141 @@
+                lambda x: not x.tax_code_id.exclude_from_registries)
+            if not taxes:
+                raise UserError(
+                    _("Invoice line %s does not have vat tax.") % line.name)
+            if len(taxes) > 1:
+                raise UserError(
+                    _("Too many vat taxes for invoice line %s.") % line.name)
+            aliquota = taxes[0].amount
+            AliquotaIVA = '%.2f' % (aliquota * 100)
+            line.ftpa_line_number = line_no
+            prezzo_unitario = self._get_prezzo_unitario(line)
+            DettaglioLinea = DettaglioLineeType(
+                NumeroLinea=str(line_no),
+                # can't insert newline with pyxb
+                # see https://tinyurl.com/ycem923t
+                # and '&#10;' would not be correctly visualized anyway
+                # (for example firefox replaces '&#10;' with space)
+                Descrizione=line.name.replace('\n', ' ').encode(
+                    'latin', 'ignore').decode('latin'),
+                PrezzoUnitario=('%.' + str(
+                    price_precision
+                ) + 'f') % prezzo_unitario,
+                Quantita=('%.' + str(
+                    uom_precision
+                ) + 'f') % line.quantity,
+                UnitaMisura=line.uos_id and (
+                    unidecode(line.uos_id.name)) or None,
+                PrezzoTotale='%.2f' % line.price_subtotal,
+                AliquotaIVA=AliquotaIVA)
+            if line.discount:
+                ScontoMaggiorazione = ScontoMaggiorazioneType(
+                    Tipo='SC',
+                    Percentuale='%.2f' % line.discount
+                )
+                DettaglioLinea.ScontoMaggiorazione.append(ScontoMaggiorazione)
+            if aliquota == 0.0:
+                if not taxes[0].kind_id:
+                    raise UserError(
+                        _("No 'nature' field for tax %s") %
+                        taxes[0].name)
+                DettaglioLinea.Natura = taxes[0].kind_id.code
+            if line.admin_ref:
+                DettaglioLinea.RiferimentoAmministrazione = line.admin_ref
+            if line.product_id:
+                if line.product_id.default_code:
+                    CodiceArticolo = CodiceArticoloType(
+                        CodiceTipo='ODOO',
+                        CodiceValore=line.product_id.default_code
+                    )
+                    DettaglioLinea.CodiceArticolo.append(CodiceArticolo)
+                if line.product_id.ean13:
+                    CodiceArticolo = CodiceArticoloType(
+                        CodiceTipo='EAN',
+                        CodiceValore=line.product_id.ean13
+                    )
+                    DettaglioLinea.CodiceArticolo.append(CodiceArticolo)
+            line_no += 1
+
+            body.DatiBeniServizi.DettaglioLinee.append(DettaglioLinea)
+
+        return True
+
+    def setDatiRiepilogo(self, invoice, body):
```

Comment:
Anche in questo caso, chiamando super eviteresti di rompere la catena di override

Reply:
con i DatiRiepilogo devo vedere se riesco a ripulirlo dalle righe create chiamando il super(), non trovo a prima vista una funzione

Reply:
Mi ricordi questo
https://github.com/OCA/l10n-italy/pull/764/files/7a5fbe0916c9d20f5c3446713e9f8ce7c5615d02#r245356248
così ho il quadro completo?

Reply:
ho tolto il super anche qui come sopra